### PR TITLE
update json config

### PR DESCRIPTION
**Pull request by alissapajer**
[ch9941]
[ch9683]

### CODE REVIEW

File: core/src/test/scala/quasar/datasource/url/UrlDatasourceConfigSpec.scala [hunk 1]
```diff
@@ -379,52 +492,129 @@ class UrlDatasourceConfigSpec extends Specification {
...
+            HeaderPaginated(UriRequest(Uri.uri("https://example4.com"), Method.GET, ""), LinkHeaderRelation.NextArchive),
+            Basic(UriRequest(Uri.uri("https://test.com"), Method.POST, """{"id": 42}"""))),
+          IMap(
+            (CIS("Accept") -> "*/*"),
+            (CIS("Authorization") -> "admin:secret")),
+          gzipped(ldjson),
+          42))
+    }
+
+    "succeeds reading correct config with data pagination and token present condition" >> {
```
Comment (rintcius) [inline core/src/test/scala/quasar/datasource/url/UrlDatasourceConfigSpec.scala]: `data pagination` -> `token pagination` - there are a couple of these left

**Comment by alissapajer** [comment]
@rintcius ready for another look. I also added one more test.

**Comment by alissapajer** [comment]
released as `13.0.2-d9d294a`

**Comment by alissapajer** [comment]
[ch10010]

**Comment by alissapajer** [comment]
@rintcius released as `13.0.5-745572b`

**Pull request merged**